updates admonitions for all pages in Infra section (#326)
polkadot-developers/polkadot-docs: infrastructure/running-a-validator/operational-tasks/general-management.md

    sudo mkdir /etc/alertmanager
    sudo nano /etc/alertmanager/alertmanager.yml
    ```

>     You must generate an [`app password` in your Gmail account](https://support.google.com/accounts/answer/185833?hl=en){target=\_blank} to allow `Alertmanager` to send you alert notification emails. Once you have the `app password`, add the following code to the configuration file to define email notifications:

Reviewer: ```suggestion
    You must generate an [app password in your Google account](https://support.google.com/accounts/answer/185833?hl=en){target=\_blank} to allow `Alertmanager` to send you alert notification emails. Once you have the app password, add the following code to the configuration file to define email notifications:
```

I don't really think we should be putting Alertmanager in back ticks, but that's a problem for another day. I'll take care of this in a separate PR

Reviewer: FYI, I created a ticket for this and started to work on it in a separate branch

    ![Live dashboard](/images/infrastructure/running-a-validator/operational-tasks/general-management/general-management-5.webp)

>  
> The [Grafana dashboards](https://grafana.com/grafana/dashboards){target=\_blank} page features user created dashboards made available for public use. For an example of available dashboards, see the [Substrate Node Metrics](https://grafana.com/grafana/dashboards/21715-substrate-node-metrics/){target=\_blank} dashboard.

Reviewer: ```suggestion
The [Grafana dashboards](https://grafana.com/grafana/dashboards){target=\_blank} page features user created dashboards made available for public use. For an example, see the [Substrate Node Metrics](https://grafana.com/grafana/dashboards/21715-substrate-node-metrics/){target=\_blank} dashboard.
```

    sudo nano /etc/alertmanager/alertmanager.yml
    ```

>     You must generate an [`app password` in your Gmail account](https://support.google.com/accounts/answer/185833?hl=en){target=\_blank} to allow `Alertmanager` to send you alert notification emails. Once you have the `app password`, add the following code to the configuration file to define email notifications:

Reviewer: ```suggestion
    Generate an [app password in your Google account](https://support.google.com/accounts/answer/185833?hl=en){target=\_blank} to enable email notifications from Alertmanager. Then, add the following code to the configuration file to define email notifications using your  email and app password: 
```

    sudo systemctl restart prometheus && sudo systemctl restart alertmanager
    ```

Now you will receive an email alert if one of your rule triggering conditions is met. 

> 
> ??? code "prometheus.yml"
> 
>         --8<-- 'code/infrastructure/running-a-validator/operational-tasks/general-management/update-prometheus.yml'

Reviewer: ```suggestion
```

This is already on line 470
